api/api/models/logs.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from api.api.models.base_models import BaseModel
+
+
+class APIUseCase(models.TextChoices):
+    """
+    Choices for the usecase field in the APILog model
+    """
+
+    NORMAL = "Normal", "Normal"
+    INTERNAL_DEMO = "Internal Demo", "Internal Demo"
+
+
+class APILog(BaseModel):
+    endpoint = models.TextField()
+    http_method = models.CharField(max_length=50)
+    use_case = models.CharField(max_length=50, choices=APIUseCase.choices)
```

Comment:
nit;

I would name this more like `source`

Reply:
Ok, I'm good with that, I tried to come up with a better name, but just ended up settling with this